lib/topological_inventory/ansible_tower/operations/order/request.rb

```diff
@@ -51,15 +66,16 @@ def order
           end
 
           def ansible_tower_client(source_id, task_id, identity)
-            Core::AnsibleTowerClient.new(source_id, task_id, identity)
+            TopologicalInventory::AnsibleTower::Operations::AnsibleTowerClient.new(source_id, task_id, identity)
```

Comment:
I do like this class naming hierarchy better than the `Core::AnsibleTowerClient` naming.